package.json

```diff
@@ -33,9 +37,19 @@
     "typescript": "~4.6.2"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.54.0",
+    "@aws-sdk/client-sfn": "^3.52.0",
+    "@middy/core": "^2.5.7",
+    "@middy/http-cors": "^2.5.7",
```

Comment:
Think we can remove this package if no longer in use

Reply:
Thanks, forgot that, removed.